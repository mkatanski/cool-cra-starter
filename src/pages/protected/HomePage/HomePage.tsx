import React from "react";

const HomeView: React.FC = () => {
  return <h1>Home page</h1>;
};

export default HomeView;
