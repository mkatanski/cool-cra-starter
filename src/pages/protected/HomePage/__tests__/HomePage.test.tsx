/* eslint-disable react/display-name */
import React from "react";
import TestRenderer from "react-test-renderer";

import HomePage from "../HomePage";

describe("pages", () => {
  describe("protected", () => {
    describe("HomePage", () => {
      it("should render HomePage component", () => {
        const tree = TestRenderer.create(<HomePage />);
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
