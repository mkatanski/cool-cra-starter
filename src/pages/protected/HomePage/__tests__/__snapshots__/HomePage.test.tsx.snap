// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages protected HomePage should render HomePage component 1`] = `
<h1>
  Home page
</h1>
`;
